repo/blob/azure/azure_storage_test.go

```diff
@@ -7,8 +7,10 @@ import (
 	"net/url"
 	"os"
 	"testing"
+	"time"
 
-	"github.com/Azure/azure-storage-blob-go/azblob"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
+	legacyazblob "github.com/Azure/azure-storage-blob-go/azblob"
```

Comment:
Separate from this PR, it may be good to refactor these tests to use the new SDK.